doc/EnvironmentVariables.md

```diff
@@ -57,7 +57,7 @@
 | Https_Proxy                             | Address of the proxy to use for outbound HTTPS requests                                                                                            | string                                                                |                               |
 | HttpSettings__Enabled                    | Whether the HTTP server should be enabled                                                                                                          | bool                                                                  | true                          |
 | HttpSettings__Port                       | The port for the HTTP protocol head to listen on                                                                                                   | int32                                                                 | 443                           |
-| HttpSettings__EnableProxyPort           | Enable the HTTP protocol head to additionally listen on port 8080. This is only reachable by internally by Docker vNet and used by ApiProxy module | bool                                                                 | true                           |
+| ProxyModuleId                           | This is the ApiProxy module name that is authorized to forward the client certificate for client CA certificate authentication | string                                                                 | iotedgeApiProxy                           |
```

Comment:
Nit: We should call this ApiProxyModuleId or ReverseProxyModuleId... Proxy is usually used for forward proxies.